src/index.js

```diff
@@ -6,6 +6,7 @@
 const common = require('./common');
 const bitgo = module.exports;
 bitgo.BitGo = require('./bitgo.js');
+bitgo.Environments = common.Environments;
```

Comment:
Why are we exporting the environments?

Reply:
In the microservice the platform runs in local mode, so it uses the local key to sign the token.

Since we use a customRootURi in the client, the sdk sets the custom key...
What I did is to get the right config depending on where is running from the sdk

const env = BitGoJS.Environments['local']; 

and then when I am about to initiate the sdk I do something like

```
params =  {
      validate: false,
      microservicesUri: this.URI,
      customRootURI: this.URI,
      serverXpub: env.serverXpub,
    };
```

Reply:
What do you think about adding something like this to the bitgo object instead of exporting from the root? This would give us a little more flexibility.
```js
getEnvironmentConfig(env) {
  return common.Environments[env || this.env] || {};
}
```

Reply:
Oh, I understand now. You need the xpub as a prerequisite for creating the bitgo object.